rubrics/tool/src/main/java/org/sakaiproject/rubrics/controller/GeneratePdfController.java

```diff
@@ -0,0 +1,268 @@
+
+        for (Criterion cri : criterion) {
+            CriterionTable = new PdfPTable(cri.getRatings().size() + 1);
+            CriterionTable.addCell(new PdfPCell(new Phrase(cri.getTitle() + " : " + getCriterionPoints(cri, evals) + " " + messageSource.getMessage("rubrics_points", null, locale) + "\n")));
+            List<Rating> ratings = cri.getRatings();
+            for (Rating rating : ratings) {
+                Paragraph ratingsParagraph = new Paragraph("", textFont);
+                ratingsParagraph.add(rating.getTitle() + " : " + rating.getPoints() + " " + messageSource.getMessage("rubrics_points", null, locale) + "\n");
+                ratingsParagraph.add(Chunk.NEWLINE);
+                Paragraph ratingsDesc = new Paragraph("", descFont);
+                if (StringUtils.isNotEmpty(rating.getDescription())) {
+                    ratingsDesc.add(rating.getDescription() + "\n");
+                }
+                ratingsParagraph.add(ratingsDesc);
+                PdfPCell newCell = new PdfPCell();
+                for (Evaluation eval : evals) {
+                    List<CriterionOutcome> outcomes = eval.getCriterionOutcomes();
+                    for (CriterionOutcome outcome : outcomes) {
+                        if (cri.getId() == outcome.getCriterionId()
+                                && rating.getId() == outcome.getSelectedRatingId()) {
+                            newCell.setBackgroundColor(BaseColor.LIGHT_GRAY);
+                            if (null != outcome.getComments() && !outcome.getComments().isEmpty()) {
+                                ratingsParagraph.add(Chunk.NEWLINE);
+                                ratingsParagraph.add(messageSource.getMessage("comments", null, locale) + Jsoup.parse(outcome.getComments()).text() + "\n");
+                            }
+                        }
+                    }
+                }
+                newCell.addElement(ratingsParagraph);
+                CriterionTable.addCell(newCell);
+                }
+            CriterionTable.completeRow();
+            document.add(CriterionTable);
+            }
+        document.close();
+        return tempFile;
+    }
+
+    private String getCriterionPoints(Criterion cri, List<Evaluation> evals) {
+
+        Integer points = 0;
+        List<Rating> ratings = cri.getRatings();
+        for (Rating rating : ratings) {
+            for (Evaluation eval : evals) {
+                List<CriterionOutcome> outcomes = eval.getCriterionOutcomes();
+                for (CriterionOutcome outcome : outcomes) {
+                    if (cri.getId() == outcome.getCriterionId()
+                            && rating.getId() == outcome.getSelectedRatingId()) {
+                        points = points + outcome.getPoints().intValue();
+                        }
+                    }
+                }
+            }
+        return points.toString();
+    }
+
+    public String getCurrentSiteId(String siteId){
```

Comment:
You mean get current site name?

Reply:
Ok. Changed for getCurrentSiteName.
Thank you.